ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,32 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :return: None
+        """
+        if len(disk.partitions) == 0:
+            pass
+
+        root_client = SSHClient(disk.storagerouter.ip, username='root')
```

Comment:
You can pass the storagerouter object to the SSHClient too, so disk.storagerouter instead of disk.storagerouter.ip
Both will work, just so you know